hw/dv/sv/flash_bkdr_util/flash_bkdr_util.core

```diff
@@ -0,0 +1,28 @@
+CAPI=2:
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:flash_bkdr_util"
+description: "Backdoor read/write flash memory for DV"
+
+filesets:
+  files_dv:
+    depend:
+      - lowrisc:opentitan:bus_params_pkg
```

Comment:
Are these dependencies all needed or do the already get pulled in from the mem_bkdr_util? Or is it just a safeguard?

Reply:
I've dropped a couple but I think the rest are all needed. I would observe that many of our core files appear not to be as minimal as they could be if they relied upon transitivity.